terraform/deployments/cluster-services/kube_prometheus_stack.tf

```diff
@@ -96,6 +97,45 @@ resource "helm_release" "kube_prometheus_stack" {
           }
         }
       }
+      serviceAccount = {
+        annotations = {
+          "eks.amazonaws.com/role-arn" = local.grafana_iam_role
+        }
+      }
+      env = {
+        "AWS_ROLE_ARN"                = local.grafana_iam_role
+        "AWS_WEB_IDENTITY_TOKEN_FILE" = "/var/run/secrets/eks.amazonaws.com/serviceaccount/token"
+        "AWS_REGION"                  = data.aws_region.current.name
+      }
+      extraSecretMounts = [
+        { name      = "aws-iam-token"
+          mountPath = "/var/run/secrets/eks.amazonaws.com/serviceaccount"
+          readOnly  = true
+          projected = {
+            defaultMode = 420
```

Comment:
It's super confusing that this is in decimal (denary). Is there way to write this literal in octal? If not, I'd suggest adding a comment with the octal mode so that others don't have to spend 5 minutes looking up what's going on 😅

Reply:
I can put the octal number in a comment.

Reply:
Cool, thanks. It looks like TF really can't do octal literals 🤯🤷